static/public/static/style.css

```
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Instrument+Sans:wght@400;500;600&display=swap");
```

Stylelint (17.12.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Instrument+Sans:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Instrument+Sans:wght@400;500;600&display=swap"" (import-notation)

(import-notation)